### PR TITLE
Configure LMDB with debug symbols

### PR DESCRIPTION
**Pull request by vpodzime**
So that we can get nice frames in backtraces including liblmdb.so.

Ticket: ENT-9198
Changelog: None

### CODE REVIEW

**Comment by vpodzime** [comment]
Build incl. exotics (NO TESTS): [Build Status](https://ci.cfengine.com/job/pr-pipeline/10392/)
Release build without exotics (NO TESTS): [Build Status](https://ci.cfengine.com/job/pr-pipeline/10393/)

File: build-scripts/install-dependencies [hunk 1]
```diff
@@ -150,8 +150,14 @@ do
     if [ x"$package_files" = x ]
     then
         # BUILD the packages
-        $BASEDIR/buildscripts/deps-packaging/pkg-build-$DEP_PACKAGING  \
-            $dep $tests $cross $optimize $debugsym $version
+        if [ "$dep" = "lmdb" ] && [ "$BUILD_TYPE" != "RELEASE" ]; then
```
Comment (craigcomstock) [inline build-scripts/install-dependencies:153]: why not just change the bits above and make debugsym=yes for all dependencies?

```
    DEBUG|CODE_COVERAGE)
        optimize=yes
        debugsym=yes
        versuffix=+untested
        tests=no;;
```

or would we need to adjust all of the .spec and rules files to make that do anything?

Comment (vpodzime) [reply build-scripts/install-dependencies:153]: Exactly.

**Review by olehermanse — APPROVED** [review]
LGTM, can I see a backtrace?

**Review by craigcomstock — APPROVED** [review]

**Comment by vpodzime** [comment]
> LGTM, can I see a backtrace?

Sure! Just use a package from this PR and use the python bindings to generate a coredump. :wink:

**Pull request merged**